curriculum/challenges/english/25-front-end-development/workshop-toggle-text-app/67cd33334277c108f5e0cb21.md

```diff
@@ -0,0 +1,115 @@
+---
+id: 67cd33334277c108f5e0cb21
+title: Step 1
+challengeType: 0
+dashedName: step-1
+demoType: onLoad
+---
+
+# --description--
+
+In the previous lecture videos, you learned about state and how to work with the `useState` hook. For this workshop, you will create an app that will hide/show a piece of text on the screen when a user clicks on a button.
+
+All of the boilerplate code, including the CSS, has been provided for you.
+
+To begin, return a `div` element inside of the `ToggleApp` component with an `id` called `toggle-container`.
+
+# --hints--
+
+You should have a `div` element.
+
+```js
+assert.exists(document.querySelector("div"));
+```
+
+Your `div` element should have an `id` called `toggle-container`.
+
+```js
+assert.exists(document.getElementById("toggle-container"));
+```
```

Comment:
````suggestion
```js
const el = document.getElementById("toggle-container"));
assert.exists(el);
assert.strictEqual(el.tagName, "DIV");
```
````

Otherwise we are only testing that *some* element has the `id`, not that it is the `div`.